packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetail.tsx

```diff
@@ -0,0 +1,105 @@
+import { Chip, CircleProgress } from '@linode/ui';
+import { Grid, styled, useTheme } from '@mui/material';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Breadcrumb } from 'src/components/Breadcrumb/Breadcrumb';
+import { ErrorState } from 'src/components/ErrorState/ErrorState';
+import { useAlertDefinitionQuery } from 'src/queries/cloudpulse/alerts';
+
+import { AlertDetailOverview } from './AlertDetailOverview';
+
+interface RouteParams {
+  /*
+   * The id of the alert for which the data needs to be shown
+   */
+  alertId: string;
+  /*
+   * The service type like linode, dbaas etc., of the the alert for which the data needs to be shown
+   */
+  serviceType: string;
+}
+
+export const AlertDetail = () => {
+  const { alertId, serviceType } = useParams<RouteParams>();
+
+  const { data: alertDetails, isError, isFetching } = useAlertDefinitionQuery(
+    Number(alertId),
+    serviceType
+  );
+
+  const generateCrumbOverrides = () => {
+    const overrides = [
+      {
+        label: 'Definitions',
+        linkTo: '/monitor/alerts/definitions',
+        position: 1,
+      },
+      {
+        label: 'Details',
+        linkTo: `/monitor/alerts/definitions/details/${alertId}`,
+        position: 2,
+      },
+    ];
+
+    return { newPathname: '/Definitions/Details', overrides };
+  };
+
+  const { newPathname, overrides } = React.useMemo(
+    () => generateCrumbOverrides(),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
+  );
```

Comment:
```suggestion
  const { newPathname, overrides } = React.useMemo(() => {
    const overrides = [
      {
        label: 'Definitions',
        linkTo: '/monitor/alerts/definitions',
        position: 1,
      },
      {
        label: 'Details',
        linkTo: `/monitor/alerts/definitions/details/${alertId}`,
        position: 2,
      },
    ];

    return { newPathname: '/Definitions/Details', overrides };
  }, [alertId]);
```

can we merge these two into one